### ISSUE TITLE
Node stoped updating

### ISSUE THREAD

**Issue by QwertyWhore** [geckogecko/storj_hoststats_app#5]
The date on the graph is still stuck in 2017 for both RT and Reputation tabs...
When synced with StorjDash the date comes current for Shared tab.

All other information is current.

**Comment by robally215** [comment]
Screenshot

**Comment by robally215** [comment]
https://imgur.com/a/R1pME

**Comment by QwertyWhore** [comment]
Screenshot , https://imgur.com/a/R1pME

**Comment by geckogecko** [comment]
@robally215 Hi thanks for creating an issue. I already noticed this bug in December 2017 and it should be fixed since then. 
Sadly you need to restart your phone **or**  deinstall and install the app- to restart the tracking of your nodes.

Let me know if you still have this issue with the current version and after a restart of your phone.

**Comment by QwertyWhore** [comment]
Hey,
 So I've been pretty keen on staying updated. I did have 1.31 installed.
The phone had been restarted over a dozen times since then. So I cleared the cache and that put the time to mid Jan. 
Then decided to just reinstall and that worked. Once installed I imported the nodes via the storjdash api and everything is current. I'll close this out, but keep an eye on it over the next few months.

Thank you again for the app.
